src/sqlite3_stubs.c

```diff
@@ -1674,6 +1674,7 @@ CAMLprim value caml_sqlite3_backup_init(
 
   if (NULL == res) raise_sqlite3_current(dst->db, "backup_init");
 
+  v_res = caml_alloc(sizeof(sqlite3_backup *), 0);
```

Comment:
This doesn't seem right: the first argument of `caml_alloc` is the number of _words_ of the newly allocated block; while `sizeof(sqlite3_backup *)` is the size of the pointer type in bytes.

You probably want `caml_alloc(1, 0)` instead.

Reply:
Yes, that's right.  Thank you.